docs/BigQueryTable-batchsink.md

```diff
@@ -53,6 +53,11 @@ is ignored if the table already exists.
 * When this is set to true, table will be created with required partition filter. 
 * When this is set to false, table will be created without required partition filter.
 
+**Clustering order**: Clustering order determines the sort order of the data. Clustering organizes data 
```

Comment:
Determines how the data is ordered
optimally-sizeD

Reply:
Done.